components/header/DashboardButton.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Link from 'next/link';
+
+import styled from 'styled-components';
+import { Routes } from '@config/routes';
+
+const DashboardLink = styled.div`
+  background: ${(props) => props.theme.color.primary[600]};
```

Comment:
You use styled-components completely right. I'm impressed that you even went through the effort and used the theme for these exercises.

Fyi there are a few utility functions that you can use to access the colors, spacing, breakpoints etc. You can find an example in this file: https://github.com/profydev/prolog-app-PaulHaze/blob/main/features/issues/components/issue-list/issue-row.tsx#L26